src/DrawOp.hpp

```diff
@@ -30,13 +30,12 @@ namespace Gosu
         
         // Number of vertices used, or: complement index of code block
         int vertices_or_block_index;
```

Comment:
I thought about removing this attribute completely but couldn't figure out how this would work then:

_DrawOpQueue.hpp:142_
```cpp
            manager.set_render_state(op.render_state);
            if (op.vertices_or_block_index >= 0) {
                op.perform(0);
            }
            else {
                // GL code
```

Reply:
I think it could be just `int block_index` and a value of -1 indicates that it is not a block, but a quad.